imap/mbexamine.c

```diff
@@ -111,6 +111,7 @@ int main(int argc, char **argv)
     char buf[MAX_MAILBOX_PATH+1];
     char *alt_config = NULL;
     int (*cb)(struct findall_data *, void *) = &do_examine;
+    int ok_count = 0;
```

Comment:
Something bugs me with that variable name. I'm sure if I read that code again in 2 years, I will have to follow the `ok_count` variable through the callbacks to understand how it is counted. The error message at the end kind of suggests it could be named `matched_mb_count` or `nmatched_mb` or the like? Off the top of my head I don't have a good suggestion.

Reply:
This is the count of things we tried and which were ok (_not_ the count of things we tried) -- the callbacks increment it on success, not on entry.  It's important for the later step, where we determine the final exit status based on a combination of "whether we successfully did anything" and "whether there were errors".

We want to be able to:

* exit success if we successfully did something (even if something else failed)
* exit with an error if we did something but it failed
* exit with a "we didn't do anything" error if we did nothing at all (which would otherwise look like a success, because of no error)

I'm not wedded to the name, but I don't have a better suggestion.

Reply:
thanks, got it